Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -1125,7 +1125,13 @@ def search(user_id, subject='', _from='', to='', before='', after='', filename='
         'v1',
         ['https://www.googleapis.com/auth/gmail.readonly'],
         command_args['userId'])
-    result = service.users().messages().list(**command_args).execute()
+    try:
+        result = service.users().messages().list(**command_args).execute()
```

Comment:
let's log the disabled mailbox details(if we have them).

Reply:
We dont have it ... the api do not mention that on the response:
HttpError: <HttpError 400 when requesting https://gmail.googleapis.com/gmail/v1/users/gsuiteroot%40wealthsimple.com/messages?q=from%3Aykornyk%40gmail.com++subject%3ATEST&alt=json&maxResults=100&includeSpamTrash=true returned "Mail service not enabled". Details: "Mail service not enabled">.
As the command returns all mailboxes we do not which one returns an error.